src/main/java/org/isf/opd/manager/OpdBrowserManager.java

```diff
@@ -63,7 +63,7 @@ protected void validateOpd(Opd opd, boolean insert) throws OHDataValidationExcep
 		Disease disease=opd.getDisease();
 		Disease disease2=opd.getDisease2();
 		Disease disease3=opd.getDisease3();
-		if (opd.getDate() == null) opd.setDate(new Date());
+		if (opd.getDate() == null) opd.setDate(new Date((new java.util.Date()).getTime()));
```

Comment:
this is old record registration date and time, it is mapped with DATETIME in the DB